specification/indices/put_index_template/IndicesPutIndexTemplateRequest.ts

```diff
@@ -178,4 +179,9 @@ export class IndexTemplateMapping {
    * @availability serverless stability=stable
    */
   lifecycle?: DataStreamLifecycle
+  /**
+   * @availability stack since=8.19.0 stability=stable
```

Comment:
While the options are technically available in `8.18.0` (which is the version you originally used in your changes), the other data stream option changes (from [00be10a](https://github.com/elastic/elasticsearch-specification/commit/00be10ac5ace4211ff41c0c640eb9b6ecaf10f07)) all specify `8.19.0`, which was probably done because the failure store feature flag (currently the only field in data stream options) was only removed in `8.19.0`, so I'm going for consistency here.